src/pages/dashboard/_StateCumulativeTestsContainer.js

```diff
@@ -104,12 +144,20 @@ export default function CumulativeTestsByStateContainer() {
     }
   `)
 
-  const data = groupAndSortStateDaily(query)
+  const [useTestsPerCapita, setUseTestsPerCapita] = useState(false)
+
+  const toggleChartData = () => setUseTestsPerCapita(u => !u)
+
+  const allData = useMemo(() => {
+    return groupAndSortStateDaily(query)
+  })
+
+  const data = useMemo(() => {
+    return useTestsPerCapita ? allData.perCapita : allData.totals
+  })
```

Comment:
same here although `useMemo` here doesn't accomplish anything because `useTestPerCapita` changes on every rerender (it's the only thing that causes a rerender)